axelrod/fingerprint.py

```diff
@@ -1,60 +1,58 @@
+"""
```

Comment:
Can we remove this please (it will become a "lie" if further changes are made that don't get recorded here). We can see the changes via the commits (and their messages can be used to in effect describe the same thing you're doing here).